components/PolygonsAndMarkers.tsx

```diff
@@ -63,6 +82,17 @@ class PolygonsAndMarkers extends Component<markersAndPolygonsProps, any> {
             title={marker.title}
             description={marker.description}
             pinColor={marker.pinColor}
+            onPress={() => {
+              location.setLatitude(marker.coordinate.latitude);
+              location.setLongitude(marker.coordinate.longitude);
+              campus.setLocation(location);
```

Comment:
why are you changing the value of the campus location, this should be a fixed value

Reply:
Will change it. I needed a campus object and decided it to pass the current location for the marker since all the markers are in a campus location by default. Ill leave it blank next commit. (ie: campus with no location or hardcoded one)

Reply:
if you look at the file in constants CampusLocations, there are the pre made campuses